### ISSUE TITLE
How to check the tensor value(this is different from issue 27519)

### ISSUE THREAD

**Issue by tu1258** [tensorflow/tensorflow#43182]
I want to check the tensor value in a specific layer and I find that tensor.numpy represents the value in eager mode. However, there is tf.placeholder in my code and tf.placeholder is not compatible with eager mode so I have to disable eager mode. Then how to check the value in this case? 

In brief, I want to see a data(eg. 1x32x32 image) going through the neural network and check all activation between every layer. It doesn't need to be training mode, inference mode is fine. Much appreciated!

**Comment by bhack** [comment]
Please ask this support question on [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow)

**Comment by jvishnuvardhan** [comment]
@tu1258 As mentioned by @bhack Please post support related issues in Stackoverflow. GitHub is mainly for bugs and performance related issues.

In TF1.x, your model runs in graph mode whereas in TF2.x Eager is by default. So, in order to access a value in a graph, you need to run a `session` (`tf.Session()`). There are lot of examples in the internet. [Here](https://cs330.stanford.edu/material/Tensorflow%20(1.x)%20Review%20Session.pdf) are couple of examples provided in this resource. Thanks!

I am closing this issue as this is resolved. Thanks!
